mix.exs

```diff
@@ -28,7 +28,7 @@ defmodule VideoRoom.MixProject do
 
   defp deps do
     [
-      {:membrane_rtc_engine, "~> 0.8.0"},
+      {:membrane_rtc_engine, github: "jellyfish-dev/membrane_rtc_engine", override: true},
       {:membrane_rtp_plugin,
```

Comment:
Please update this plugin as well.

BTW, you can review the last added commit: https://github.com/membraneframework/membrane_rtp_plugin/commit/aa197acda401fb282a5c5be8d11acc45c4575212